dao/src/main/java/org/thingsboard/server/dao/tenant/TenantProfileServiceImpl.java

```diff
@@ -112,7 +111,8 @@ public void deleteTenantProfile(TenantId tenantId, TenantProfileId tenantProfile
         this.removeTenantProfile(tenantId, tenantProfileId, false);
     }
 
-    private void removeTenantProfile(TenantId tenantId, TenantProfileId tenantProfileId, boolean isDefault) {
```

Comment:
A private method is never responsible for a transaction. You need to find the public method that calls this private method and make those transactional:
![image](https://user-images.githubusercontent.com/79898499/185866160-54fbbbcc-fc13-4f7e-9208-966ee8b40300.png)
![image](https://user-images.githubusercontent.com/79898499/185866234-d75aac8e-26a8-4f16-b3b3-16294b9e4298.png)

![image](https://user-images.githubusercontent.com/79898499/185866005-eccc1142-c7e7-4216-8651-5565e628d587.png)